services-js/commissions-search/src/ResultList.js

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from "react";
+
+import gql from "graphql-tag";
+import { Query } from "react-apollo";
+
+class ResultList extends Component {
+  render() {
+    var GET_RESULTS;
+
+    // need to construct query based on policy ids
+    // if the facet array is empty then send null else send array of keys
+    if (
+      Object.keys(this.props.facets) === undefined ||
+      Object.keys(this.props.facets).length === 0
+    ) {
+      GET_RESULTS = gql`{
+          commissions(query: "${
+            this.props.keywords
+          }", policyTypeIds: ${null}, hasOpenSeats: ${this.props.seats}) {
+            id
+            name
+            policyType {
+              name
+            }
+            homepageUrl
+          }
+        }`;
+    } else {
+      GET_RESULTS = gql`{
+          commissions(query: "${
+            this.props.keywords
+          }", policyTypeIds: [${Object.keys(
+        this.props.facets
+      )}], hasOpenSeats: ${this.props.seats}) {
+            id
+            name
+            policyType {
+              name
+            }
+            homepageUrl
+          }
+        }`;
+    }
+
+    function Result(props) {
+      // Check for null policyType
+      if (!props.policyName) {
+        return (
+          <li className="n-li">
+            <a
+              className="n-li-b n-li-b--r n-li-b--c n-li-b--fw n-li--in g g--mt0"
+              href={props.homepageUrl}
+            >
+              <div className="n-li-t g--8">{props.name}</div>
+              <div className="n-li-ty n-li-ty--r g--44" />
+            </a>
+          </li>
+        );
+      } else {
+        return (
+          <li className="n-li">
+            <a
+              className="n-li-b n-li-b--r n-li-b--c n-li-b--fw n-li--in g g--mt0"
+              href={props.homepageUrl}
+            >
+              <div className="n-li-t g--8">{props.name}</div>
+              <div className="n-li-ty n-li-ty--r g--44">
+                {props.policyName.name}
+              </div>
+            </a>
+          </li>
+        );
+      }
+    }
+
+    const ResultComponents = () => (
+      <Query query={GET_RESULTS}>
+        {({ loading, error, data }) => {
+          if (loading) return "Loading...";
+          if (error) return `Error! ${error.message}`;
+
+          if (data.commissions.length === 0) {
+            return (
+              <div className="b-c b-c--mh">
+                <h2 className="h2 m-t000 m-b300">No Results Found</h2>
+                <div className="intro-text supporting-text lh--200">
+                  Thomas Paine noted "These are the times that try men's souls."
```

Comment:
Use curlies: “ ” ’

Reply:
how?

Reply:
Option-[ and Shift-Option-[ for “ and ”, Shift-Option-] for ’